### DISCUSSION TITLE
When using K3S, the request https://10.43.0.1:443/apis/apps/v1/deploymens will report dial tcp 10.43.0.1:443: i/o timeout

### DISCUSSION THREAD

**Discussion by Xxm-YX** [k3s-io/k3s#7522] [Q&A]
Environmental Info:
K3s Version:
```
root@linux:~# kubectl version
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.15+k3s1", GitCommit:"c7a91e1ccdace64eae4a41b9dc15ef8d7aed6d23", GitTreeState:"clean", BuildDate:"2022-01-25T00:24:22Z", GoVersion:"go1.15.15", Compiler:"gc", Platform:"linux/arm64"}
Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.15+k3s1", GitCommit:"c7a91e1ccdace64eae4a41b9dc15ef8d7aed6d23", GitTreeState:"clean", BuildDate:"2022-01-25T00:24:22Z", GoVersion:"go1.15.15", Compiler:"gc", Platform:"linux/arm64"}
```
Node(s) CPU architecture, OS, and Version:
```
Linux linux 5.10.66 #77 SMP Web Apr 5 21:38:10 CST 2023 aarch64 aarch64 aarch64 GNU/Linux
```
![bb9f2748120dbb8be16ed5122186be15](https://github.com/k3s-io/k3s/assets/61789235/866ad059-4953-4b60-838f-d0729dedd391)

The full log file is below
[k3s (2).log](https://github.com/k3s-io/k3s/files/11437664/k3s.2.log)

I hope you can help me to find out where the problem is, I really don't know where to start, thank you very much

**Comment by brandond** [comment]
v1.20.15+k3s1 is a very old and end of life release. Please try again with a newer version.

**Reply by Xxm-YX** [reply]
Thank you very much, I am going to try to use the new version. I have tried v1.23.6+k3s1 version before, and this problem still occurs

**Reply by brandond** [reply]
You haven't said what application you're deploying that these logs are coming from? These are not K3s log messages and I have no idea what you're trying to do.

I also see a bunch of errors in your logs about missing iptables modules; have you run `k3s check-config` ?
```
May 09 10:58:29 linux k3s[2694]: E0509 10:58:29.518980    2694 proxier.go:1598] Failed to execute iptables-restore: exit status 2 (iptables-restore v1.8.4 (legacy): Couldn't load target `REJECT':No such file or directory
May 09 10:58:29 linux k3s[2694]: Error occurred at line: 5
May 09 10:58:29 linux k3s[2694]: Try `iptables-restore -h' or 'iptables-restore --help' for more information.
May 09 10:58:29 linux k3s[2694]: )
May 09 10:58:29 linux k3s[2694]: I0509 10:58:29.519176    2694 proxier.go:879] Sync failed; retrying in 30s
```
